cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -616,6 +619,276 @@ async def _get_trustworthiness_score_async(
+
+        if quality_preset in {"high", "best"}:
+            raise ValidationError(
+                f"Invalid quality preset: {quality_preset}. TLMHybrid only supports 'low' and 'medium' presets."
+            )
+        self._score_quality_preset = quality_preset
+
+        if options is not None:
+            validate_tlm_hybrid_score_options(options)
+        self._score_options = options
+        self._prompt_options = cast(
+            TLMOptions, get_tlm_hybrid_prompt_options(self._score_options, self._response_model)
+        )
+
+        self._timeout = timeout  # TODO: are we handling this any differently?
+        self._verbose = verbose
+
+        self.tlm_response = TLM(
+            self._api_key,
+            quality_preset="base",
+            options=self._prompt_options,
+            timeout=self._timeout,
+            verbose=self._verbose,
+        )
+
+        self.tlm_score = TLM(
+            self._api_key,
+            quality_preset=self._score_quality_preset,
+            options=self._score_options,
+            timeout=self._timeout,
+            verbose=self._verbose,
+        )
+
+    def prompt(
+        self,
+        prompt: Union[str, Sequence[str]],
+    ) -> Union[TLMResponse, List[TLMResponse]]:
+        """
+        Gets response and trustworthiness score for any text input.
+
+        This method prompts one LLM with the given prompt(s), producing completions (like a standard LLM)
+        and then uses a second LLM to perform an evaluation of the given response and provides a trustworthiness scores quantifying the quality of the output.
+
+        Args:
+            prompt (str | Sequence[str]): prompt (or list of multiple prompts) for the language model.
+                Providing a batch of many prompts here will be faster than calling this method on each prompt separately.
+        Returns:
+            TLMResponse | List[TLMResponse]: [TLMResponse](#class-tlmresponse) object containing the response and trustworthiness score.
+                If multiple prompts were provided in a list, then a list of such objects is returned, one for each prompt.
+                This method will raise an exception if any errors occur or if you hit a timeout (given a timeout is specified).
+                Use it if you want strict error handling and immediate notification of any exceptions/timeouts.
+
+                If running this method on a big batch of prompts: you might lose partially completed results if TLM fails on any one of them.
+                To avoid losing partial results for the prompts that TLM did not fail on,
+                you can either call this method on smaller batches of prompts at a time
+                (and save intermediate results between batches), or use the [`try_prompt()`](#method-try_prompt) method instead.
+        """
+        prompt_response = self.tlm_response.prompt(prompt)
+
+        if isinstance(prompt_response, list):
+            assert isinstance(prompt, Sequence)
```

Comment:
why is there a test here?

Reply:
This was required for mypy to pass. Because the input prompt arg can either be a sequence or str, so we needed to make sure that it was of the right type for each case.

Updated these to be explicit if/else statements.